ui-core/src/participant/dashboard/Alert.tsx

```diff
@@ -0,0 +1,89 @@
+import { IconDefinition } from '@fortawesome/fontawesome-svg-core'
+import { faCheck, faCircleInfo, faTriangleExclamation, faXmark } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import classNames from 'classnames'
+import React from 'react'
+import { InlineMarkdown } from '../landing/Markdown'
+
+export type AlertLevel =
+  | 'primary'
+  | 'info'
+  | 'success'
+  | 'warning'
+  | 'danger'
+
+export type AlertTrigger =
+  'NO_ACTIVITIES_REMAIN' |
+  'WELCOME' |
+  'STUDY_ALREADY_ENROLLED'
+
+export type ParticipantDashboardAlert = {
+  id?: string
+  title: string
+  detail?: string
+  type: AlertLevel
+  trigger: AlertTrigger
+}
+
+export const alertDefaults: Record<AlertTrigger, ParticipantDashboardAlert> = {
```

Comment:
something to keep in the back of our minds as we tackle the full multi-study dashboard is moving as much default text to the database as we can, so that we can change it and internationalize it more easily